ansible/roles/nginx-proxy/templates/openshift/nginx-deployment.yml

```diff
@@ -67,6 +67,8 @@ spec:
             - name: nginx-htpasswd
               mountPath: "{{ nginx_basic_user_dir_path }}"
               readOnly: true
+            - name: nginx-logs
+              mountPath: "{{ nginx_proxy_log_dir }}"
```

Comment:
The definition of the `nginx-logs` volume is still missing from the `volume` section above